Documentation/cli-prerequisites.md

```diff
@@ -0,0 +1,81 @@
+CLI native prerequisites
+=========================
+
+This document outlines the dependencies needed to run .NET Core CLI tools. Most of these dependencies are also .NET Core's general dependencies, so installing them will make sure that you can run applications written for .NET Core in general.
+
+## Windows dependencies
+On Windows, the only dependency is the VC++ Redistributable. Depending on the version of Windows you are running on, the versions are changing.
+
+* Windows 10
+    * [Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145)
+* Windows 7 & 8, Windows Server 2008 & 2012
+    *  [Visual C++ Redistributable for Visual Studio 2012 Update 4](https://www.microsoft.com/en-us/download/confirmation.aspx?id=30679)
```

Comment:
Sorry, my point was that while we chain the VC 2015 Redist into the bundle, we don't chain in VC 2012.  Is this actually still a requirement on pre Windows 10?  Can't everything be on VC 2015 (and if not, should we be installing VC 2012 on Win 7-8.1 machines when oh use the bundle?)